Index: net-snmp-5.7.2~dfsg/agent/snmp_agent.c
===================================================================
--- net-snmp-5.7.2~dfsg.orig/agent/snmp_agent.c	2015-12-09 11:09:33.000000000 +0000
+++ net-snmp-5.7.2~dfsg/agent/snmp_agent.c	2015-12-09 11:11:08.602815263 +0000
@@ -3044,18 +3044,6 @@
                             "request response %d out of range\n",
                             request->index));
                 /*
-                 * I'm not sure why inclusive is set unconditionally here (see
-                 * comments for revision 1.161), but it causes a problem for
-                 * GETBULK over an overridden variable. The bulk-to-next
-                 * handler re-uses the same request for multiple varbinds,
-                 * and once inclusive was set, it was never cleared. So, a
-                 * hack. Instead of setting it to 1, set it to 2, so bulk-to
-                 * next can clear it later. As of the time of this hack, all
-                 * checks of this var are boolean checks (not == 1), so this
-                 * should be safe. Cross your fingers.
-                 */
-                request->inclusive = 2;
-                /*
                  * XXX: should set this to the original OID? 
                  */
                 snmp_set_var_objid(request->requestvb,
